tests/idris2/reflection010/run

```diff
@@ -0,0 +1,3 @@
+$1 --no-banner --check Name.idr
```

Comment:
As for test `reflection009` here the flags `--no-color --console-width 0` are needed to avoid system-dependent differences in the test output.

Reply:
oops I see thank you, guess I haven't looked at test runners in the last couple months :>